packages/odyssey-react/.storybook/translations/en.json

```diff
@@ -0,0 +1,28 @@
+{
+  "variant.primary": "Primary",
+  "variant.secondary": "Secondary",
+  "variant.danger": "Danger",
+  "variant.clear": "Clear",
+  "variant.dismiss": "Dismiss",
+  "variant.neutral": "Neutral",
+  "variant.success": "Success",
+  "variant.caution": "Caution",
+  "variant.wide": "Wide",
+  "label.fieldLabel": "Field label",
+  "label.statusLabel": "Status label",
+  "label.hint": "This is a field hint.",
+  "label.error": "Invalid error description.",
+  "label.optional": "Optional",
+  "label.descriptor": "Descriptor",
+  "label.item": "Item",
+  "input.searchValue": "Jupiter",
+  "input.searchDefaultValue": "Search Planets",
+  "number.one": "One",
+  "number.two": "Two",
+  "number.three": "Three",
+  "position.top": "Top",
+  "position.right": "Right",
+  "position.bottom": "Bottom",
+  "position.left": "Left",
+  "position.label": "{position} tooltip label"
+}
```

Comment:
Maintaining this is sort of tedious. I would be interested in finding a way to automate the translation to supported locales in the future.